docs/docs/development/documentation.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Documentation
+sidebar_label: Documentation
+---
+
+:::note
+The documentation is written with [docusaurus](https://docusaurus.io/). You don't need install anything specific for docusaurus but it can be helpful to reference their documentation if needed.
+:::
+
+:::note
+If you are working with the documentation from within VSCode+Docker please be aware the documentation will not be auto-generated when making changes while the server is running. You'll n eed to restart the server when saving changes to the documentation.
+:::
+
+## Preparation
+
+In order to view and verify documentation updates locally you'll need to install `node.js` and `npm`. Please see [nodejs.org](https://nodejs.org/) for details on how to install these tools.
+
+## Verifying Documentation Updates Locally
+
+To verify documentation updates locally, follow the following procedure.
+
+1. Navigate to the `docs` folder
+2. Run `npm ci` to build the necessary tools if you haven't run it before or the zmk sources were updated
+3. Run `npm start` to start the local server that will let you see your documentation updates via a web browser
+4. Open a web browser and navigate to `http://localhost:3000` to view your changes
```

Comment:
At least when running it natively, `npm start` will automatically launch a browser to this address. Maybe prefix this with something like "If it doesn't open automatically, ..."